### ISSUE TITLE
01_init migration failure in some configurations

### ISSUE THREAD

**Issue by Maxime-J** [umami-software/umami#1261]
Now that there is no need to import an sql file at install,
there can be an issue with a fresh install and some MySQL servers which don't have InnoDB set by default.

With that error:
```
Migration name: 01_init

Database error code: 1071

Database error:
Specified key was too long; max key length is 1000 bytes

Please check the query number 1 from the migration file.
```
`InnoDB` was specified in the old way sql file https://github.com/umami-software/umami/blob/master/sql/schema.mysql.sql
Not in the init migration

Don't know if you want to handle that, as it shouldn't be very common nowadays,
and changing an existing migration isn't recommended https://www.prisma.io/docs/concepts/components/prisma-migrate

**Comment by mikecao** [comment]
We need to update that. InnoDB needs to be set.

**Comment by mikecao** [comment]
Fixed in v1.34.0.
